src/schema/artwork/index.js

```diff
@@ -446,16 +446,9 @@ export const artworkFields = () => {
     },
     is_purchasable: {
       type: GraphQLBoolean,
-      description: "True for inquireable artworks that have an exact price.",
-      resolve: artwork => {
-        return (
-          !has_editions(artwork.edition_sets) &&
-          is_inquireable(artwork) &&
-          isExisty(artwork.price) &&
-          !has_price_range(artwork.price) &&
-          artwork.forsale
-        )
-      },
+      deprecationReason:
+        "Purchase requests are not supported. Replaced by buy now.",
+      resolve: () => null,
```

Comment:
Maybe better to resolve `false` rather than `null`? But since this is typed as a nullable boolean, this should be 👍 as well.

Reply:
My thinking was that it is not used - so it's unknown... But I guess in gravity it defaults to false, so maybe that would be better to keep it false for consistency. Actually maybe just return the value - it'll make it possible to view historical inquiries correctly in MP. Should I change? @mzikherman ?